qemu/tests/cfg/nvdimm.cfg

```diff
@@ -53,3 +53,16 @@
             ndctl_install_cmd = "rpm -q ndctl || yum install -y ndctl"
             create_dax_cmd = "ndctl create-namespace -m dax -e namespace0.0 -f -v -a 4096"
             del_dax_cmd = "ndctl create-namespace -m memory -e namespace0.0 -f -v -a 4096"
+        - nvdimm_mapsync:
+            type = nvdimm_mapsync
+            no Host_RHEL.m7
+            pmem_mem = on
+            nv_backend = nvdimm0
+            mount_dir = "/mnt/pmem"
+            mem-path = "${mount_dir}/${nv_backend}"
+            format_command = "mkfs.xfs -f ${dev_path} -m reflink=0"
+            mount_command = "mkdir -p ${mount_dir} && mount -o dax ${dev_path} ${mount_dir}"
+            truncate_command = "truncate -s ${size_mem} ${mem-path}"
```

Comment:
Seems the `${mem-path}` is not the expect mem-path '/mnt/pmem/nvdimm0'. It does not support double escape.

Reply:
It works well on my host. The file is created successfully.

Reply:
@huangyum  boot vm with the mem-path can create the `nvdimm0` too.
Could you try sleep after step ` process.run(truncate_cmd)`,  then check `nvdimm0` in host ?
In my side, no file. But after start vm, there is.

Reply:
You are right, updated.